en/docs/graphql-api.md

```diff
@@ -0,0 +1,90 @@
+# GraphQL APIs
+
+Explore how you can easily design, develop, and manage GraphQL APIs in Choreo to implement your business API strategy.
+
+??? note "Enable GraphQL components"
```

Comment:
Shall we mention the GQL is in preview mode in the note title? 

Example: `How to enable GraphQL component Feature Preview` or `Enable the Feature Preview for GraphQL components`